src/app/InteractionModelEngine.cpp

```diff
@@ -367,6 +367,12 @@ bool InteractionModelEngine::SubjectHasActiveSubscription(const FabricIndex aFab
     return isActive;
 }
 
+bool InteractionModelEngine::SubjectHasPersistedSubscription(const FabricIndex & aFabricIndex, const NodeId & subject)
+{
+    // TODO : Implement persisted sub verification
```

Comment:
please file a ticket on it, thanks

Reply:
Ticket already existed; added the issue number